File: src/IsbInterop/Tasks/Wrappers/TaskInfo.vb
Namespace Tasks.Wrappers
  ''' <summary>
  ''' Обертка над ITaskInfo.
  ''' </summary>
  Friend Class TaskInfo
    Inherits CustomWorkInfo
    Implements ITaskInfo
    ''' <summary>
    ''' Конструктор.
    ''' </summary>
    ''' <param name="rcwITaskInfo">COM-объект ITaskInfo.</param>
    Public Sub New(rcwITaskInfo As Object)
      MyBase.New(rcwITaskInfo)
    End Sub
  End Class
End NameSpace

File: src/IsbInterop.CSharp/Tasks/Wrappers/TaskInfo.cs
namespace IsbInterop.Tasks.Wrappers
{
  /// <summary>
  /// Обертка над ITaskInfo.
  /// </summary>
  internal class TaskInfo : CustomWorkInfo, ITaskInfo
  {
     /// <summary>
    /// Конструктор.
    /// </summary>
    /// <param name="rcwITaskInfo">COM-объект ITaskInfo.</param>
    public TaskInfo(object rcwITaskInfo) : base(rcwITaskInfo) { }
  }
}
